src/pages/LandingPage/Signup/Signup.js

```diff
@@ -37,57 +37,67 @@ const Signup = () => {
     }
   };
   return (
-    <div className="signup-container">
-      <h1 className="signup-title">SIGN UP</h1>
-      <form className="display-flex-column">
-        <div className="display-flex-row">
-          <label className="form-label">Email</label>
-          <input
-            type="email"
-            class="form-field"
-            placeholder="shubham@gmail.com"
-            value={userSignUp.email}
-            onChange={(e) => {
-              setUserSignUp({ ...userSignUp, email: e.target.value });
-            }}
-          />
-        </div>
-        <div className="display-flex-row">
-          <label className="form-label">Password</label>
-          <input
-            type="password"
-            className="form-field"
-            placeholder="Enter new password"
-            value={userSignUp.password}
-            onChange={(e) => {
-              setUserSignUp({ ...userSignUp, password: e.target.value });
-            }}
-          />
-        </div>
-        <div className="display-flex-row">
-          <label className="form-label">Confirm Password</label>
-          <input
-            type="password"
-            className="form-field"
-            placeholder="Re-type your password"
-            value={userSignUp.confirmPassword}
-            onChange={(e) => {
-              setUserSignUp({ ...userSignUp, confirmPassword: e.target.value });
-            }}
-          />
-        </div>
+    <div className="wrapper">
+      <div className="signup-container">
+        <h1 className="signup-title">SIGN UP</h1>
+        <form className="display-flex-column">
+          <div className="display-flex-row">
+            <label className="form-label">Email</label>
+            <input
+              type="email"
+              class="form-field"
```

Comment:
You can use react className attribute here

Reply:
yes, I just forgot to update it, thanks for pointing it out.